lib/model/migrations/20241028-01-add-force-entity-def-source.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 ODK Central Developers
```

Comment:
Note to self: change this migration date if merging *after* https://github.com/getodk/central-backend/pull/1253/files (`20241029-01-schedule-entity-form-upgrade-create-forms.js`)

Reply:
(done)